package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alpinejs-money",
-  "version": "1.1.3",
+  "version": "1.2.0",
```

Comment:
I wouldn't say this is a minor release, more of a patch.

Reply:
Since it changes current behaviour, it can also be seen as minor

Reply:
I disagree. It's more of a patch to the existing code than a change people should be aware of.